client/data/marketplace/use-marketplace-reviews.ts

```diff
@@ -12,14 +12,14 @@ const reviewsApiBase = '/sites/marketplace.wordpress.com/comments';
 const reviewsApiNamespace = 'wp/v2';
 const queryKeyBase: QueryKey = [ 'marketplace-reviews' ];
 
-type ProductType = 'plugin' | 'theme';
+export type ProductType = 'plugin' | 'theme';
 
 type ProductProps = {
 	productType: ProductType;
-	pluginSlug: string;
+	productSlug: string;
```

Comment:
Can we use `slug` instead of `productSlug`? The item reviewed might not be a product, eg .org theme.

Reply:
In this file, the word `Product` is used all over the place referring to `theme` or `plugin`. To confirm my understanding of this specific field, we want to avoid confusion with the `product_slug` from the `store_products` table. Is that right?

Reply:
I have renamed `productSlug` as `slug` as part of https://github.com/Automattic/wp-calypso/pull/84480/commits/4dc8f76831c4bedc1e7183c8bb6862c399a3fe64, we can always rename the other occurrences of the prefix `product` if some other entities are added. Currently, I like the slightly more specific `productType` and `productName` than the more generic `type` or `name`, but please let me know if you would prefer to do it as part of these changes.